<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAccountDialog</class>
 <widget class="QDialog" name="EditAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit account</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="vlMain">
   <item>
    <layout class="QHBoxLayout" name="hlContent">
     <item>
      <layout class="QVBoxLayout" name="vlContent1">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="eName"/>
       </item>
       <item>
        <widget class="QLabel" name="labelDescription">
         <property name="text">
          <string>Description (optional):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="eDescription"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vlContent2">
       <item>
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbType">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCurrency">
         <property name="text">
          <string>Currency:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCurrency"/>
       </item>
       <item>
        <widget class="QLabel" name="labelInitialBalance">
         <property name="text">
          <string>Initial balance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sbInitialBalance">
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMinimalBalance">
         <property name="text">
          <string>Minimal balance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sbMinimalBalance">
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vsContent2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="lineMain">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlFooter">
     <item>
      <spacer name="hsFooter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>EditAccountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
